src/main/java/Database.java

```diff
@@ -0,0 +1,47 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Database {
```

Comment:
Consider adding a couple of class-level comments describing the class.
Might not be immediately clear otherwise what this `Database` class is intended for.